html/global_attributes.json

```diff
@@ -911,6 +919,10 @@
       "hidden": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Global_attributes/hidden",
+          "spec_url": [
+            "https://html.spec.whatwg.org/multipage/interaction.html#the-hidden-attribute",
+            "https://html.spec.whatwg.org/multipage/rendering.html#hiddenCSS"
```

Comment:
I know someone put this second link in the MDN page but why is it important to authors? Not sure I would have added it here or on MDN.

Reply:
Agree

> I know someone put this second link in the MDN page but why is it important to authors? Not sure I would have added it here or on MDN.

Agreed — so dropped the second one in e9068f6